spec/services/like_service_spec.rb

```diff
@@ -115,20 +175,83 @@
 
     it "returns all likes of a post" do
       post = alice.post(:status_message, text: "hello", public: true)
-      likes = [alice, bob, eve].map {|user| LikeService.new(user).create(post.id) }
+      likes = [alice, bob, eve].map {|user| LikeService.new(user).create_for_post(post.id) }
 
       expect(LikeService.new.find_for_post(post.id)).to match_array(likes)
     end
   end
 
+  describe "#find_for_comment" do
+    context "with user" do 
+      it "returns likes for a public post comment" do
+        post = alice.post(:status_message, text: "hello", public: true)
+        comment = CommentService.new(bob).create(post.id, "Hello comment")
+        like = LikeService.new(alice).create_for_comment(comment.id)
+        expect(LikeService.new(eve).find_for_comment(comment.id)).to include(like)
+      end
+
+      it "returns likes for visible private post comments" do
+        comment = CommentService.new(bob).create(post.id, "Hello comment")
+        like = LikeService.new(alice).create_for_comment(comment.id)
+        expect(LikeService.new(bob).find_for_comment(comment.id)).to include(like)
+      end
+
+      it "doesn't return likes for a posts comment the user can not see" do
+        expect {
+          LikeService.new(eve).find_for_comment(alice_comment.id)
+        }.to raise_error ActiveRecord::RecordNotFound
+      end
+
+      it "returns the user's like first" do
+        post = alice.post(:status_message, text: "hello", public: true)
+        comment = CommentService.new(alice).create(post.id, "I like my own post")
+
+       [alice, bob, eve].map {|user| LikeService.new(user).create_for_comment(comment.id)}
+       [alice, bob, eve].each do |user|
+          expect(
+            LikeService.new(user).find_for_comment(comment.id).first.author.id
+          ).to be user.person.id
+        end
```

Comment:
Layout/BlockAlignment: `end` at 214, 8 is not aligned with `[alice, bob, eve].each do |user|` at 210, 7.